src/components/Picklist/utils.js

```diff
@@ -18,7 +18,7 @@ export function insertChildOrderly(children, child, nodes) {
 
 export function getChildMenuItemNodes(ref) {
     if (ref) {
-        return ref.querySelectorAll('a[role="menuitem"]');
+        return ref.querySelectorAll('div[role="option"]');
```

Comment:
Since we don't get aware of this bug we need to find somehow test this scenario.